bitwarden_license/src/Commercial.Core/AdminConsole/Providers/RemoveOrganizationFromProviderCommand.cs

```diff
@@ -67,27 +67,31 @@ public class RemoveOrganizationFromProviderCommand : IRemoveOrganizationFromProv
 
         await _organizationRepository.ReplaceAsync(organization);
 
-        var customerUpdateOptions = new CustomerUpdateOptions
+        if (!string.IsNullOrEmpty(organization.GatewayCustomerId) &&
+            !string.IsNullOrEmpty(organization.GatewaySubscriptionId))
```

Comment:
This seems like a common check - could it be moved to a getter on `Organization`?

Reply:
Moved it to one of our Core extension files: https://github.com/bitwarden/server/pull/3999/commits/2dad72a5c3ae826afff592278639c26431081a8e